iothub/device/src/InternalClient.cs

```diff
@@ -1887,6 +1889,24 @@ internal void ValidateModuleTransportHandler(string apiName)
 
         #endregion Module Specific API
 
+        #region Convention driven operations
```

Comment:
Would a partial class be better here as well?

Reply:
yeah, we could do that, sure